projects/ADuCM3029_demo_cn0326/src/AD7793.c

```diff
@@ -224,7 +228,9 @@ void AD7793_Calibrate(uint8_t ui8channel, enMode mode)
     AD7793_WriteRegister(AD7793_REG_MODE, ui32reg_value);
 
     /* Wait until RDY bit from STATUS register is high */
-    while ((AD7793_ReadRegister(AD7793_REG_STAT) & RDY_BIT) == RDY_BIT);
+    while ((AD7793_ReadRegister(AD7793_REG_STAT) & RDY_BIT) == RDY_BIT &&
+    		timeout < 10000)
+    	timeout++;
 }
```

Comment:
It is a bit uncertain about how long it takes to reach the timeout. Maybe we should consider implementing this more like in this example: https://github.com/analogdevicesinc/linux/blob/rpi-4.9.y/drivers/iio/dac/ad5758.c#L254
Also, how long does it actually take for the RDY_BIT to be set? Do we have a worst case scenario? Is this information available in the datasheet?

Reply:
But, I think for now we can remove the timeout completely and add fix it later with a different PR